opendevin/runtime/runtime.py

```diff
@@ -58,50 +38,53 @@ class Runtime:
     sid: str
     file_store: FileStore
 
-    def __init__(
-        self,
-        event_stream: EventStream,
-        sid: str = 'default',
-        sandbox: Sandbox | None = None,
-    ):
+    def __init__(self, event_stream: EventStream, sid: str = 'default'):
         self.sid = sid
-        if sandbox is None:
-            self.sandbox = create_sandbox(sid, config.sandbox.box_type)
-            self._is_external_sandbox = False
-        else:
-            self.sandbox = sandbox
-            self._is_external_sandbox = True
-        self.browser: BrowserEnv | None = None
-        self.file_store = InMemoryFileStore()
         self.event_stream = event_stream
         self.event_stream.subscribe(EventStreamSubscriber.RUNTIME, self.on_event)
+        atexit.register(self.close_sync)
+
+    async def initialize(self) -> None:
```

Comment:
Like with the other class, maybe rename to init_async (or ainit), just an idea.

Reply:
renamed to `ainit` :D